source/community/governance.md

```diff
@@ -0,0 +1,71 @@
+
+- the Fortran-lang GitHub organization and can invite members, give write-access to projects, and create teams
+- the Fortran-lang Discourse and can give moderator privileges as well as moderate the forum
+- the fortran-lang.org domain
+- the [@fortranlang](https://twitter.com/fortranlang) Twitter account
+
+- the Fortran-lang [YouTube](https://www.youtube.com/@fortranprogramminglanguage7454) channel
+
+- the Fortran-lang [Mailing-list](https://groups.io/g/fortran-lang)
+
+Maintainers and contributors should contact the administrators in case any action for those assets is required.
+
+Current administrators are
+
+- Ondřej Čertík ([@certik](https://github.com/certik), [ondrej@certik.us](mailto:ondrej@certik.us))
+
+- Milan Curcic ([@milancurcic](https://github.com/milancurcic), [milancurcic@hey.com](mailto:milancurcic@hey.com))
+
+- Sebastian Ehlert ([@awvwgk](https://github.com/awvwgk), [awvwgk@disroot.org](mailto:awvwgk@disroot.org))
+
+- Laurence Kedward ([@lkedward](https://github.com/lkedward), [laurence.kedward@bristol.ac.uk](mailto:kedward@bristol.ac.uk))
+
+- Giannis Nikiteas ([@gnikit](https://github.com/gnikit), [giannis.nikiteas@gmail.com](mailto:giannis.nikiteas@gmail.com))
+
+:::{div} sd-text-left sd-fs-2 sd-font-weight-bold sd-text-primary
+Maintainers
+:::
+
+The maintainers are responsible for organizing the core projects of Fortran-lang, like the package manager, the standard library, and the webpage. Maintainers are responsible for
+
+- coordinating (larger) projects and steering long-term plans
+- helping new contributors to get acquainted with the project and provide guidance for their first contribution
+- guiding new reviewers to provide comments and feedback for code contributions in pull requests and patches
+- keeping the issue tracker and project boards organized
+- making releases and determining their content
+
+Unless specially required, all discussions and activities of the maintainers will be public and made in collaboration with the community and contributors. Private communications or decisions made by the maintainers will be summarized to the community and contributors publicly after removing personal, private, or sensitive information, that should not be posted publicly on the internet.
+
+:::{div} sd-text-left sd-fs-2 sd-font-weight-bold sd-text-primary
+Contributors
+:::
+
+Everybody is welcome to contribute to Fortran-lang and its projects. Providing and discussing ideas as well as developing concepts and designs are integral contributions to any project. All contributors can steer and add to the long-term vision of the Fortran-lang projects.
+
+:::{div} sd-text-left sd-fs-2 sd-font-weight-bold sd-text-primary
+MIT License
+:::
+
+To the extent possible under law, the authors have waived all copyright and related or neighboring rights to the Fortran-lang community governance document, as per the CC-0 public domain dedication.
```

Comment:
Can we change the license to MIT as Most of the Fortran - Lang projects are under MIT License.

Reply:
I'm sorry, but I'm having trouble understanding your specific requirements. Could you please provide further clarification what you're looking for?

Reply:
we cannot use CC-0 public domain license , Most of the Fortran - Lang projects are under MIT license.

Reply:
I believe it is now correct

Reply:
I think either is fine. MIT is more common for software whereas CC-* licenses are more common for docs.

Reply:
> the authors have waived all copyright and related or neighboring rights to
as Most of the Fortran - Lang projects are under MIT license , this is an incorrect statement.

Reply:
Could someone kindly advise me on what actions I should take?  I have no knowledge about licenses 😅

Reply:
it seems to be good for the docs.